jenkins/ci.suse.de/ardana-trigger-trackupstream.yaml

```diff
@@ -0,0 +1,81 @@
+- job:
+    name: 'ardana-trigger-trackupstream'
+
+    triggers:
+      - gerrit:
+          server-name: 'gerrit.suse.provo.cloud'
+          trigger-on:
+            - change-merged-event
+            - ref-updated-event
```

Comment:
What is this event responsible for?

Reply:
A push, that may have been done without a review.

Reply:
Removing it in: https://github.com/SUSE-Cloud/automation/pull/2790